src/config/constants/index.ts

```diff
@@ -97,7 +97,7 @@ export const getConstants = memoize((): Constants => {
     FANTOM_PROVIDER_HTTPS: 'https://rpc.ftm.tools',
     ARBITRUM_PROVIDER_HTTPS: 'https://arb1.arbitrum.io/rpc',
     CONTRACT_ADDRESSES: {
-      zapIn: '0x645ab9Eb0Bd26a8a6d9aad4F4B4809C4069bc9D5',
+      zapIn: '0x8e52522e6a77578904ddd7f528a22521dc4154f5',
```

Comment:
Please update to checksummed address

Reply:
fixed on b897bd76a89b4b063778b3b22833d96e29538987